sts-trace-ot/src/main/java/stackstate/opentracing/STSSpanContext.java

```diff
@@ -12,6 +14,23 @@
 import stackstate.trace.api.STSTags;
 import stackstate.trace.common.sampling.PrioritySampling;
 
+class STSSpanContextPidProvider implements ISTSSpanContextPidProvider {
+
+  @Override
+  public long getPid() {
+    String processName = java.lang.management.ManagementFactory.getRuntimeMXBean().getName();
+    return Long.parseLong(processName.split("@")[0]);
```

Comment:
I like to see some comment here why we split on the '@', or alternatively: this implementation is probably based on some StackOverflow post or example, I'd like to see a link to that as a way of documentation.